week-1/K-functions-parameters/exercise2.js

```diff
@@ -1,5 +1,7 @@
-// Declare your function first
+function divide(a, b) {
+  return a / b;
+}
 
-var result = divide(3, 4);
+var result = divide((a = 3), (b = 4));
```

Comment:
This line was fine, not need to change it. When we call the function by saying `divide(3, 4)`, the program knows that `3` is `a` and `4` is `b` because you defined it that way on line 1.